block-management/block-holder.js

```diff
@@ -33,8 +33,8 @@ export default class BlockHolder extends React.Component<PropsType, StateType> {
 	}
 
 	getBlockForType() {
-		if ( this.props.blockType === 'code' ) {
-			const blockType = getBlockType( 'core/code' );
+		const blockType = getBlockType( this.props.blockType );
+		if ( blockType ) {
 			const Code = blockType.edit;
```

Comment:
Looks good, nitpicking comment: can we replace `const Code` to a more generic `const Block`?

Reply:
Took the opportunity (256ab18) to generalize a bit more than just the renaming, by passing all the props to the block. Let me know what you think @maxme .